development/tools/Makefile

```diff
@@ -1,5 +1,5 @@
 resolve:
-	dep ensure -v -vendor-only
+	go mod verify
```

Comment:
Can we change the Makefile to be in general more like e.g. this one: https://github.com/kyma-incubator/hydroform/blob/master/Makefile